### DISCUSSION TITLE
ジャンパのランド剥がれとトラックボールの動作不良

### DISCUSSION THREAD

**Discussion by Zeni-Y** [Yowkees/keyball#192] [Q&A] [answered]
重ねての質問申し訳ありません．

keyball44において，トラックボールが動作しません（他のキーは通常通り動作します）．

ジャンパ部分のランドが剥がれてしまってしまったので，回路図を参照して，トラックボール側のMISOとPro MicroのMISOはつなげましたが，トラックボールが動作しません．
（ #168  を参考にしました．）
![S__53911557](https://user-images.githubusercontent.com/37218752/215987444-41496c57-c373-4db7-9bc1-76875f0c2a7b.jpg)

![S__53911559](https://user-images.githubusercontent.com/37218752/215987069-414af939-aedb-4329-9306-c5a05b05e3a1.jpg)

その他に， #51 での内容を一通りためしましたが，動作しません．

また不安な点として，電源をつけた状態で一度keyball基板をLGコンスルーに差し込む際に，表裏逆にはめ込んでしまいました．
keyball基板を差し込むと赤い光が1秒程度付くのですが，これは正常に動作しているということなのでしょうか？

付きまして，自分の対応（トラックボール側のMISOとPro MicroのMISOの接続）が正しいのかと，他にすべき対応がありましたら教えて頂けると幸いです．

お忙しいところ恐縮ですが，ご回答よろしくお願い致します．

**Comment by Zeni-Y** [comment, marked as answer]
改めて各ジャンパ部分のはんだ付けをし直したところ，keyball基板の赤い光が常につくようになり，正常にトラックボールの操作ができるようになりました．

ご心配おかけして申し訳ありません．

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありませんでした。
ジャンパの半田付けが原因でしたか。意外なところでした。
無事にトラックボールが動き、安心しました！
今後ともどうぞよろしくお願いいたします。
